etc/picongpu/hydra-hzdr/default.tpl

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+# Copyright 2013-2017 Axel Huebl, Anton Helm, Rene Widera
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for hydra PBS batch system
+
+#PBS -q !TBG_queue
+#PBS -l walltime=!TBG_wallTime
+# Sets batch job's name
+#PBS -N !TBG_jobName
+#PBS -l nodes=!TBG_nodes:ppn=!TBG_coresPerNode
+#PBS -l mem=256GB
```

Comment:
QUESTION:
Do you gibe memory per node (or per core)? From slum I know, that per core is used.

Reply:
in PBS it should be per node, otherwise this should not have been able to be scheduled on the available resources (the ones he mentions in the PR description)

Reply:
in [this ref](https://www.osc.edu/supercomputing/batch-processing-at-osc/pbs-directives-summary) it's *for all nodes* ^^

Reply:
Thanks for the reference - there it states "all memory across all nodes" thus this should be multiplied with the number of nodes requested. This only ensures 256GB/node nodes if the node number is below or equal to 4. With higher node numbers, we could land on a 64GB/node machine. 😱

Reply:
@psychocoderHPC Please multiply this number with the number of nodes requested.